src/Identification/Queue/Events/Processing.php

```diff
@@ -87,9 +88,9 @@ public function __construct(JobProcessing $event)
      */
     private function unserializeToJob(string $object)
     {
-        $stdClassObj = preg_replace('/^O:\d+:"[^"]++"/', 'O:'.strlen('stdClass').':"stdClass"', $object);
+        $stdClassObj = preg_replace('/^O:\d+:"[^"]++"/', 'O:'.strlen(TenancyJob::class).':"'.TenancyJob::class.'"', $object);
 
-        return unserialize($stdClassObj);
+        return unserialize($stdClassObj, ['allowed_classes' => [TenancyJob::class]]);
```

Comment:
This causes PHP to not unserialize the ModelIdentifiers that would cause the connections